automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -28,9 +28,33 @@ main() {
     echo "Deploy KubeVirt latest stable release"
     ./hack/deploy-kubevirt.sh
 
+    echo "Get skip tests regex"
```

Comment:
Please add a comment or change this message, to explain we need to skip tests that check the opposite KMP webhook mode

Reply:
I made it  so that `__get_skipped_tests` is general, so I'll add a specific comment inside it
Done

Reply:
Please add short explanation in this line why we need to skip test of the opposite KMP webhook mode, something like 
"Run only tests of the current configured KMP webhooks mode (opt-in/opt-out)"

Reply:
`__get_skipped_tests` is meant to be generic, so a generic comment is ok here imo.

Reply:
you could also add this link https://github.com/k8snetworkplumbingwg/kubemacpool#kubemacpool-service

Reply:
Yes I saw that you add those echos its is great 
I think we should add comment about skipping tests, since its not straight forward to understand why we need to skip in this case
And its good reminder for our future self's why we did it :)

Reply:
i am against comments, the code is / should be self explanatory.
i wont block, its up to you two

Reply:
>I think we should add comment about skipping tests, since its not straight forward to understand why we need to skip in this case
And its good reminder for our future self's why we did it :)

but why isn't the current echo's enough?

Reply:
@oshoval We discussed this few times now, I know what is you opinion about comments and generally I agree, but there are cases when you need more and explain why you need to do something
like in this case.

@RamLavi The echos explains what are we doing, I think we need to add comment about why we need to do this.
It took me a few minutes to understand why we need this and I am familiar with KMP code, for someone who doesn't it will be hard to understand it in my opinion.

Reply:
@ormergi this is why the commit message is so important.
We can't explain everything in the code, but we make sure that anyone who want to further understand a change - could do so by reading the commit message (via blame or github GUI)

do you think the commit message needs further clarification?

Reply:
I understand, in this case it would be much better to explain shortly why is it necessary to skip specific tests instead of the current echo message.
Since this is a bash script its not documenting itself well like other languages.

For the commit message, I would change the subject to `kmp functests, skip redundant KMP webhook mode tests`
and for the body, 
```
KMP functests includes tests for both opt-in and opt-out KMP webhook modes
and since CNAO deploys KMP with opt-in mode,
it is necessary to skip tests of the opposite mode (opt-out tests).

This commit adds a function that checks what is the current configuration of KMP in the cluster
 and skip the relevant tests accordingly.
```

Reply:
Done. 
redundant is not the correct wording, but change to fit

Reply:
Thanks